src/components/ButtonGroup/ButtonGroup.scss

```diff
@@ -48,51 +46,24 @@ $item-spacing: spacing(tight);
     &:not(:first-child) {
       margin-left: -(border-width());
     }
-
-    // stylelint-disable-next-line selector-max-combinators
-    > * {
-      border-radius: 0;
-    }
-
-    // stylelint-disable-next-line selector-max-combinators
-    &:first-child > * {
-      border-top-left-radius: border-radius();
-      border-bottom-left-radius: border-radius();
-    }
-
-    // stylelint-disable-next-line selector-max-combinators
-    &:last-child > * {
-      border-top-right-radius: border-radius();
-      border-bottom-right-radius: border-radius();
-    }
   }
 
   .Item-focused {
     z-index: z-index(focused, $stacking-order);
   }
-}
-
-.fullWidth {
-  .Item {
-    flex: 1 1 auto;
 
-    // stylelint-disable-next-line selector-max-combinators
-    > * {
-      @include button-full-width;
+  &.globalTheming {
+    .Item {
+      // stylelint-disable-next-line selector-max-class, selector-max-specificity
+      &:not(:first-child) {
+        margin-left: spacing(extra-tight);
```

Comment:
Looks like we need to add a new variable at the top for this margin value like [`$segmented-spacing: rem(2px);`](https://www.figma.com/file/4dAAt5iFPSaxUKiYVKrkYj/DL%E2%80%93Polaris-(Web)%3A-Components?node-id=2330%3A262) since `spacing(extra-tight)` is `rem(4px)`.

Reply:
Could have sworn this was 4px. I changed it now and was able to remove the .'globalTheming' class.